controller_report_xls/controllers/main.py

```diff
@@ -30,9 +30,10 @@
 import xlwt
 from cssutils import parseString
 from lxml import etree
+from werkzeug import url_decode
+
 from odoo.addons.report.controllers import main
 from odoo.addons.web.http import request, route
-from werkzeug import url_decode
```

Comment:
This is autoformatting, it should be in a dedicated commit where `pre-commit-vauxoo` autofixes are applied. Besides, this could be applied in the previous version.

Same for other autoformatting changes.